desktop/core/src/desktop/lib/rest/raz_http_client_test.py

```diff
@@ -31,22 +31,21 @@ class TestRazHttpClient():
 
   def test_get_file(self):
     with patch('desktop.lib.rest.raz_http_client.AdlsRazClient.get_url') as raz_get_url:
-      with patch('desktop.lib.rest.raz_http_client.HttpClient.execute') as http_execute:
+      with patch('desktop.lib.rest.raz_http_client.HttpClient.execute') as raz_http_execute:
 
         raz_get_url.return_value = {
           'token': 'sv=2014-02-14&sr=b&sig=pJL%2FWyed41tptiwBM5ymYre4qF8wzrO05tS5MCjkutc%3D&st=2015-01-02T01%3A40%3A51Z&se=2015-01-02T02%3A00%3A51Z&sp=r'
         }
-        http_execute.return_value = 'my file content'
+        raz_http_execute.return_value = 'my_file_content'
 
         client = RazHttpClient(username='test', base_url='https://gethue.blob.core.windows.net')
-        f = client.execute(http_method='GET', path='/gethue/data/customer.csv')
+        f = client.execute(http_method='GET', path='/gethue/data/customer.csv', params={'action': 'getStatus'})
```

Comment:
Nicer test now!

Reply:
And later we could test the PUT or read() which add new headers
https://github.com/cloudera/hue/blob/master/desktop/libs/azure/src/azure/abfs/abfs.py